base/applications/regedit/childwnd.c

```diff
@@ -363,6 +423,12 @@ LRESULT CALLBACK ChildWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lPa
                                                     IMAGE_ICON, 12, 12, 0);
         SendMessageW(g_pChildWnd->hAddressBtnWnd, BM_SETIMAGE, IMAGE_ICON, (LPARAM)g_pChildWnd->hArrowIcon);
 
+        if (SUCCEEDED(CoCreateInstance(&CLSID_AutoComplete, NULL, CLSCTX_INPROC_SERVER, &IID_IAutoComplete, (void**)&pAutoComplete)))
+        {
+            pAutoComplete->lpVtbl->Init(pAutoComplete, g_pChildWnd->hAddressBarWnd, (IUnknown*)&g_DummyEnumStrings, NULL, NULL);
+            pAutoComplete->lpVtbl->Release(pAutoComplete);
```

Comment:
I wonder whether you could use the slightly more "elegant" helper macros (that should be in the autogenerated headers)
```suggestion
            IAutoComplete_Init(pAutoComplete, g_pChildWnd->hAddressBarWnd, (IUnknown*)&g_DummyEnumStrings, NULL, NULL);
            IAutoComplete_Release(pAutoComplete);
```